external/src/main/java/com/redhat/cloud/custompolicies/engine/process/Receiver.java

```diff
@@ -31,16 +32,23 @@
     @ConfigProperty(name = "engine.receiver.store-events")
     boolean storeEvents;
 
+    @ConfigProperty(name = "engine.receiver.log-events", defaultValue = "false")
+    boolean logEvents;
+
     @Inject
     SystemProfileClient client;
 
     @Inject
     AlertsService alertsService;
 
+    @Timed(name = "engine-incoming", absolute = true)
     @Incoming("kafka-hosts")
     public CompletionStage<Void> processAsync(JsonObject json) {
         String tenantId = json.getString(TENANT_ID);
         String insightsId = json.getString(INSIGHT_ID_FIELD);
+        if (logEvents) {
+            System.out.printf("Got msg for tenant [%s] and insightsId [%s] \n ",  tenantId , insightsId);
```

Comment:
This should use log.debugf()

Reply:
Well. In container world that all goes to stdout, so I am not sure if that is really needed.

Reply:
It is necessary for the behavior of async processing (System.out isn't async).